wp-content/plugins/core/src/Blocks/Types/Hero.php

```diff
@@ -0,0 +1,71 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Blocks\Types;
+
+use Tribe\Gutenpanels\Blocks\Block_Type_Interface;
+use Tribe\Gutenpanels\Blocks\Sections\Content_Section;
+use Tribe\Gutenpanels\Blocks\Sections\Sidebar_Section;
+use Tribe\Project\Blocks\Block_Type_Config;
+
+class Hero extends Block_Type_Config {
+
+	public const NAME = 'tribe/hero';
+
+	public const BACKGROUND_IMAGE = 'bg-image';
+	public const ICON             = 'icon';
+	public const TITLE            = 'title';
+	public const DESCRIPTION      = 'description';
+	public const CTA              = 'cta';
+
+	public function build(): Block_Type_Interface {
+		return $this->factory->block( self::NAME )
+			->set_label( __( 'Hero', 'tribe' ) )
+			->add_class( 'test-hero' )
```

Comment:
clean up all test classes being passed to the edit template, was just for testing porpoises

Reply:
Yes will do, once we get this merged, I'll update this whole thing ; )